packages/manager/modules/dbaas-logs/src/logs/detail/index/logs-index.service.js

```diff
@@ -1,96 +1,106 @@
+    this.$http = $http;
     this.$q = $q;
     this.$translate = $translate;
+    this.iceberg = iceberg;
     this.CucCloudPoll = CucCloudPoll;
     this.CucControllerHelper = CucControllerHelper;
     this.LogsHelperService = LogsHelperService;
+    this.CucUrlHelper = CucUrlHelper;
     this.CucServiceHelper = CucServiceHelper;
     this.LogsConstants = LogsConstants;
-    this.LogsHomeService = LogsHomeService;
-    this.LogsOrderService = LogsOrderService;
-    this.IndexApiService = OvhApiDbaas.Logs()
-      .Index()
-      .v6();
-    this.IndexAapiService = OvhApiDbaas.Logs()
-      .Index()
-      .Aapi();
-    this.OperationApiService = OvhApiDbaas.Logs()
-      .Operation()
-      .v6();
     this.newIndex = {
       description: '',
       alertNotifyEnabled: false,
       nbShard: this.LogsConstants.NB_SHARD_MIN,
     };
   }
 
-  getAccountDetails(serviceName) {
-    return this.LogsHomeService.getAccountDetails(serviceName);
+  getNewIndex() {
+    return this.newIndex;
   }
 
-  getOrderCatalog(ovhSubsidiary) {
-    return this.LogsOrderService.getOrderCatalog(ovhSubsidiary);
+  getIndices(serviceName) {
+    return this.iceberg(`/dbaas/logs/${serviceName}/output/opensearch/index`)
+      .query()
+      .expand('CachedObjectList-Pages')
+      .limit(10000)
+      .execute().$promise;
   }
 
-  getNewIndex() {
-    return this.newIndex;
+  getPaginatedIndices(
+    serviceName,
+    offset = 0,
+    pageSize = 25,
+    sort = { name: 'name', dir: 'desc' },
+    filters = null,
+  ) {
+    let res = this.iceberg(`/dbaas/logs/${serviceName}/output/opensearch/index`)
+      .query()
+      .expand('CachedObjectList-Pages')
+      .limit(pageSize)
+      .offset(offset)
+      .sort(sort.name, sort.dir);
+    if (filters !== null) {
+      filters.forEach((filter) => {
+        res = res.addFilter(filter.name, filter.operator, filter.value);
+      });
+    }
+    return res.execute().$promise.then((response) => ({
+      data: response.data,
+      meta: {
+        totalCount:
+          parseInt(response.headers['x-pagination-elements'], 10) || 0,
+      },
+    }));
   }
 
-  getIndicesIds(serviceName) {
-    return this.IndexApiService.query({ serviceName }).$promise;
+  getIndiceIds(serviceName) {
+    return this.iceberg(`/dbaas/logs/${serviceName}/output/opensearch/index`)
+      .query()
+      .execute()
+      .$promise.then((response) => {
+        return response.data;
+      });
   }
 
-  getIndices(serviceName) {
-    return this.getIndicesIds(serviceName)
-      .then((indices) => {
-        const promises = indices.map((indexId) =>
-          this.getIndexDetails(serviceName, indexId),
-        );
-        return this.$q.all(promises);
-      })
+  getIndicesForAlias(serviceName, aliasId) {
+    return this.$http
+      .get(
+        `/dbaas/logs/${serviceName}/output/opensearch/alias/${aliasId}/index`,
+      )
       .catch((err) =>
         this.LogsHelperService.handleError('logs_index_get_error', err, {}),
```

Comment:
I see 'logs_index_get_error' used in multiple places, can you please move this to constant ?

Reply:
It looks that it has been always written like this. I'm not sure the purpose of the current PR is to tackle this subject.